Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -128,7 +164,8 @@ protected virtual ICollection<string> FilesToImport(object parameter)
 
                 var fileNames = dialog.FileNames;
                 var fileExtensions = fileNames.Select(Path.GetExtension);
-                if (fileExtensions.Any(fileExt => !_importableExtensions.Contains(fileExt)))
+                var importableExtensions = ImportableExtensions;
```

Comment:
why the local reference copy?

Reply:
I think this was used more than once before. 
BTW, it is actually not a reference copy since the property returns a new collection each time.